src/Nox.Generator/Application/Dto/EntityCreateDto.template.cs

```diff
@@ -93,9 +93,9 @@ public abstract class {{className}}Base : IEntityDto<DomainNamespace.{{entity.Na
     /// {{entity.Name}} {{relationship.Description}} {{relationship.Relationship}} {{relationship.EntityPlural}}
     /// </summary>
     {{- if relationship.Relationship == "ZeroOrMany" || relationship.Relationship == "OneOrMany"}}
```

Comment:
```suggestion
    {{- if relationship.WithMultiEntity}}
```

or maybe align with the UpdateDto and use the relationship.WithSingleEntity condition and in the else generate the list property.

Reply:
Updated